packages/lumx-core/src/scss/components/tabs/lumapps/_index.scss

```diff
@@ -119,3 +119,21 @@
         @include lumx-tabs-link-active(lumx-base-const('theme', 'DARK'));
     }
 }
+
+/* Tab panel
+   ========================================================================== */
+
+.#{$lumx-base-prefix}-tab-panel {
+    display: none;
+    outline: none;
+
```

Comment:
I get where you are coming from, but this is an antipattern accessibility whise...
Having a focusable element without an outline is probably as confusing for the user as not having the element focusable.

If we don't like the "focused" style, it should be improved not removed.

Reply:
It's the anti pattern used on the whole LumX design system xD
We remove the focus tab on click focus and add it back on tab focus

Reply:
Have you tried it ? It doesn't work on my end, there is no outline ring on tab focus. :( 
It comes back when I remove this line though.

Reply:
It was the style I used that was bad :/ 
It should work now

Reply:
Nice! Sorry about the initial comment, I spoke too soon and misread!